spec/test_app/db/schema.rb

```diff
@@ -47,6 +47,7 @@
     t.integer  "group_id"
     t.string   "banner_url"
     t.text     "sermon_notes"
+    t.string   "audio_url"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.